<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QProductionPaint</class>
 <widget class="QWidget" name="QProductionPaint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="ProductionTabList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>180</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Team 1</string>
    </attribute>
    <widget class="QFrame" name="ProductionList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>140</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGraphicsView" name="object1_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object1_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object1_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object1_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object1_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object1_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Team 2</string>
    </attribute>
    <widget class="QFrame" name="ProductionList_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>140</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGraphicsView" name="object2_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object2_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object2_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object2_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object2_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="object2_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>75</y>
        <width>60</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
